lms/djangoapps/commerce/signals.py

```diff
@@ -91,220 +83,3 @@ def get_request_user():
     """
     request = RequestCache.get_current_request()
     return getattr(request, 'user', None)
-
-
-def _process_refund(refund_ids, api_client, course_product, is_entitlement=False):
```

Comment:
Why move the location of these?  Could you just call them without being a signal?  The location should not matter.

Reply:
As we discussed, they are no longer used by just the Signal so it felt cleaner to have them in utils.  I feel that importing signals defeats the purpose of the Django Signal method.